addons/drawbuild/functions/fnc_drawBuildZeus.sqf

```diff
@@ -14,92 +14,88 @@ Starts the selection handler to select multiple points for you to draw
 
 params [["_pos",[0,0,0],[[]],3], ["_unit",objNull,[objNull]]];
 
-GVAR(lastPole) = objNull;
-
-//create display 
-if (!createDialog "crowsza_drawbuild_display") exitWith {["Failed to open drawbuild dialog"] call EFUNC(main,showHint)};
-
-//get display
-private _display = uiNamespace getVariable "crowsza_drawbuild_display";
-
-//use ZENs method to init display posisions. I can use it as I should have same base setup as ZEN uses in their gui
-[_display] call zen_common_fnc_initDisplayPositioning;
-
-// save position that is selected 
-// _display setVariable ["crowsza_drawbuild_pos", _pos];
-
-// array of "options" to build with
-private _arrOptions = [
-	"Land_HBarrier_3_F",					//hesco default
-	"Land_HBarrier_Big_F",					//big hesco default
-	"Land_BagFence_Short_F",				//sandbag wall - default
-	"Land_SandbagBarricade_01_F",			//tall sandbags
-	"Land_TyreBarrier_01_line_x4_F",		//tire wall
-	"Land_ConcreteWall_01_m_4m_F", 			//concrete wall
-	
-	//tanoa sandbags
-	"Land_HBarrier_01_line_3_green_F",		//tanoa hesco (green)
-	"Land_HBarrier_01_big_4_green_F",		//big tanoa hesco 
-	"Land_BagFence_01_short_green_F",		//tanoa sandbag wall
-	"Land_Mil_WallBig_4m_F",				//military wall
-	"Land_Fortress_01_5m_F",				//land fortress wall 5m
-
-	//fences
-	"Land_Hedge_01_s_2m_F",					//grass hedge
-	"Land_NetFence_02_m_4m_F",				//net fence
-	"Land_New_WiredFence_5m_F",				//wire fence
-	"Land_Razorwire_F",						//razor wire
-
-	//misc.
-	"PowerCable_01_StraightLong_F",			//power cable
-	"Land_PowerLine_03_pole_F", 			//concrete overhead line
-	"Land_PowerLine_02_pole_small_F"		//wood overhead line
-];
-
-// only add grad trenches if that mod is loaded
-private _hasGradTrench = isClass (configFile >> "CfgPatches" >> "grad_trenches_main");
-if (_hasGradTrench) then {
-	_arrOptions = _arrOptions + ["fort_envelopebig"]
-};
-
-//display all items
-{
-    private _mainItem = _x;
-
-	// picture and name
-    private _picture = getText(configfile >> "CfgVehicles" >> _mainItem >> "editorPreview");
-    private _name = getText(configfile >> "CfgVehicles" >> _mainItem >> "displayName");
-
-    private _tooltip = format ["%1", _name];
-    // private _tooltip = format ["%1\n%2", _name, _mainItem];
-
-	// put in object
-	private _spot = IDC_ICON_GRID_FIRST + _forEachIndex;
-	// get spot
-	private _ctrlSpot = _display displayCtrl _spot;
-
-	// set picture and tooltip
-	_ctrlSpot ctrlSetText _picture;
-	_ctrlSpot ctrlSetTooltip _tooltip;
-
-	// handler for click on icon
-	[_ctrlSpot, "ButtonClick", {
-            params ["_ctrlSpot"];
-            _thisArgs params ["_object", "_display"];
+// TODO: sort by display name? Or by category?
```

Comment:
possible you can give screenshot of how each mode looks? 
Might be easiest to find stuff alphabetically?

Reply:
We'd need to define the "categories" ourselves, and there's a few ways we could decide to do it. E.g. "sandbags", "walls", "misc.", etc.; or "urban", "rural", "military", etc.

Without some sort of dummy "header" items in the list, it might not be particularly clear/obvious how or why they're sorted that way.

I think I'm leaning towards alphabetical sorting

Reply:
yeah, I would go alphabetical sorting. What is obviously a wall to us, might not be for someone else. But most tend to know about the item name from general zeusing. So should be able to find it.